public/js/controllers/home-controller.js

```diff
@@ -120,4 +121,71 @@ function($scope, $http, planService, notificationService) {
             });
         });
     };
+
+    $scope.hasPrereqs = function(course) {
+
+        //This is ugly
```

Comment:
I agree, I was about to say that the commenting thing is purely aesthetic and not worth the effort. Don't worry about being stern, that's the point of these reviews and process, we look at things, we make decisions, and we uphold them later.